CMakeLists.txt

```diff
@@ -4,30 +4,27 @@
 cmake_minimum_required(VERSION 3.22.1)
 
 # project name and language
-project(axie_tflite LANGUAGES CXX)
-
-set(TENSORFLOW_SOURCE_DIR $ENV{TENSORFLOW_SOURCE_DIR})
-set(ANDROID_NDK_ROOT $ENV{ANDROID_NDK_ROOT})
-
-if (NOT DEFINED ANDROID_NDK_ROOT)
-    message(FATAL_ERROR  "NDK root path is not defined. Please use cmake -ANDROID_NDK_ROOT=/path/to/android_ndk")
-endif()
+project(whisperax LANGUAGES CXX)
```

Comment:
nit: 
let's use c++23, may require some code changes
we'll also turn on warnings as errors